src/components/Table/TableToolbar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Material } from '@balena/ui-shared-components';
+import { alpha } from '@mui/material/styles';
```

Comment:
We should not be importing directly from this package since it's not a direct dependency of autoui. Is it already included in the `Material` export from ui-shared? If not, can we add the export in the ui-shared PR and change this import to be from ui-shared?